test/factory.test.js

```diff
@@ -280,19 +295,29 @@ describe('Factory', function () {
       });
     });
 
-    it('can run more complicated webpack builds with multiple output files', function (done) {
+    it('can run more complicated webpack builds with multiple output files, minify and write to disk', function (done) {
       const data = config('other');
 
       this.timeout(5000);
       data.entry = 'webpack.config.js';
-
-      run(new Factory(data, webpackworker.run), function (error, factory) {
+      const fact = new Factory(data, webpackworker.run);
+      run(fact, function (error, factory) {
         if (error) return done(error);
 
         assume(Object.keys(factory.output)).to.have.length(4);
         assume(Object.keys(factory.compressed)).to.have.length(4);
 
-        done();
+        //
+        // This tests the last bits where we minify as well as write to disk
+        //
+        fact.minify((err) => {
+          assume(err).is.falsey();
```

Comment:
would be good to add a few assertions here wrt stored content of the uglifyjs output on `fact`.